lib/spandex_phoenix.ex

```diff
@@ -235,17 +241,35 @@ defmodule SpandexPhoenix do
     :erlang.raise(kind, reason, stack)
   end
 
-  # Private Helpers
+  @doc false
+  def trace_request(conn, opts) do
```

Comment:
errrrrrr it looks like i'm no longer using this in my code and should put it back where i grabbed it from